Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed an issue in `RebindingUISample` preventing UI navigation without Keyboard and Mouse present.
 - Fixed an issue in `RebindActionUI` which resulted in active binding not being shown after a scene reload. ISXB-1588.
 - Fixed an issue in `GamepadIconExample` which resulted in icons for left and right triggers not being displayed after a rebind to the exact same controls. ISXB-1593.
+- Fixed the compilation warnings when used with Unity 6.4 (ISX-2349)
```

Comment:
Nitpick: Missing full-stop (period) at the end of the sentence.

Reply:
```suggestion
- Fixed the compilation warnings when used with Unity 6.4 (ISX-2349).
```